diff --git a/conf_mng.c b/conf_mng.c
index aebf262..fd33d40 100644
--- a/conf_mng.c
+++ b/conf_mng.c
@@ -111,7 +111,7 @@
 
 #define ASTERISK_CONF_EXTENSIONS_FROM_EXTERNAL \
     "[from-external]\n" \
-    "exten => %s,1,Dial(PJSIP/%s)\n" \
+    "exten => %s,1,Dial(PJSIP/%s&PJSIP/%s)\n" \
     "\n"
 
 #define ASTERISK_CONF_EXTENSIONS_FROM_INTERNAL \
@@ -393,7 +393,7 @@ static int conf_write_extensions(sip_conf_t *sip_conf)
     /* Configure incoming call to ring on first extension */
     if (sip_conf->enabled && sip_conf->username != NULL &&
         fprintf(file, ASTERISK_CONF_EXTENSIONS_FROM_EXTERNAL,
-        sip_conf->username, ASTERISK_EXTENSION_1) < 0)
+        sip_conf->username, ASTERISK_EXTENSION_1, ASTERISK_EXTENSION_2) < 0)
     {
         log_message(LERR, "Failed to write %s\n",
             ASTERISK_CONF_PATH_EXTENSIONS);
